src/main/java/com/openlattice/chronicle/constants/MessageOutcome.kt

```diff
@@ -0,0 +1,15 @@
+package com.openlattice.chronicle.constants
+
+import com.openlattice.chronicle.data.MessageType
+import java.time.OffsetDateTime
+import java.util.*
+
+data class MessageOutcome(
+    val messageType: MessageType,
+    val dateTime: OffsetDateTime,
+    val participantId: String,
+    val url: String,
+    var isSuccess: Boolean,
+    val sid: String,
+    val studyId :UUID,
+)
```

Comment:
I think this belongs in chronicle-api like with `MessageDetails`. In the main stack, all of these types of classes live in the api project, think `EntitySet`, `Organization`, etc.

Reply:
it's not used by anything in chronicle-api, but I can move it there if you like